avm/res/insights/private-link-scope/main.bicep

```
@@ -241,161 +253,44 @@ output privateEndpoints array = [
   for (pe, i) in (!empty(privateEndpoints) ? array(privateEndpoints) : []): {
```

```suggestion
  for (item, index) in (privateEndpoints ?? []): {
```
Whether you use `i` oder `index` is of course up to you